k8s/service_controller/model/service_instance.go

```diff
@@ -22,9 +22,9 @@ type LastOperation struct {
 }
 
 type CreateServiceInstanceRequest struct {
```

Comment:
Even a step further, the SB model should be defined in a separate folder, e.g.,

service_controller/model/...
service_broker/model/...

Since the broker model is really the model for the broker's API.